src/Microsoft.AspNet.Authorization/AuthorizationPolicy.cs

```diff
@@ -96,7 +99,9 @@ public static AuthorizationPolicy Combine(AuthorizationOptions options, IEnumera
                 {
                     foreach (var authType in authTypesSplit)
                     {
-                        policyBuilder.AuthenticationSchemes.Add(authType);
+                        if (string.IsNullOrEmpty(authType))
```

Comment:
Instead of this explicit check for empty, can you use the the overoad of Split which takes in `StringSplitOptions.RemoveEmptyEntries`